docs/source/en/using-diffusers/other-formats.md

```diff
@@ -240,6 +240,50 @@ Benefits of using a single-file layout include:
 1. Easy compatibility with diffusion interfaces such as [ComfyUI](https://github.com/comfyanonymous/ComfyUI) or [Automatic1111](https://github.com/AUTOMATIC1111/stable-diffusion-webui) which commonly use a single-file layout.
 2. Easier to manage (download and share) a single file.
 
+### DDUF
+
+<Tip warning={true}>
+
+DDUF is an experimental file format and APIs related to it can change in the future.
+
+</Tip>
```

Comment:
Prefer using this syntax style now since its shorter/easier :)

```suggestion
> [!WARNING]
>DDUF is an experimental file format and APIs related to it can change in the future.
```